pkg/kubelet/root_context_linux.go

```diff
@@ -33,3 +33,7 @@ func (kl *Kubelet) getRootDirContext() (string, error) {
 	// Get the SELinux context of the rootDir.
 	return selinux.Getfilecon(kl.getRootDir())
 }
+
```

Comment:
I'm cool with this going into these files for now, but we will probably wind up renaming this file to something like `selinux_linux.go` shortly.